cmd/snap/cmd_buy_test.go

```diff
@@ -72,15 +76,23 @@ func (s *buyTestMockSnapServer) checkCounts() {
 	}
 }
 
-func (s *SnapSuite) TestBuyHelp(c *check.C) {
+func (s *BuySnapSuite) SetUpTest(c *check.C) {
```

Comment:
Its probably better to call the parent suite here, i.e. `s.SnapSuite.SetUpTest(c)` as the first line, same for teardown.